src/documents/ComponentConfigurationOptions.md

```diff
@@ -260,59 +247,62 @@ See also: [Component Grades](ComponentGrades.md)
   </tr>
   <tr>
     <th>See also</th>
-    <td><a href="FrameworkConcepts.md#model-objects">Model Objects</a><br/>
-        <a href="ChangeApplierAPI.md">ChangeApplier API</a><br/>
-        <code><a href="#applier">applier</a></code><br/>
-        <code><a href="#changeapplieroptions">changeApplierOptions</a></code></td>
+    <td><a href="OptionsMerging.md">Options Merging</a></td>
   </tr>
 </table>
 
-### applier ###
+### `dynamicComponents` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>A <a href="ChangeApplier.md">ChangeApplier</a> object for the model provided with the <code><a href="#model">model</a></code> option.</td>
+    <td>An object containing named definitions of the component's dynamic subcomponents. Rather than exactly one subcomponent being associated with its parent from these records, there may be one subcomponent per element of an array, or one per firing of an event.</td>
```

Comment:
This reference to 'dynamic subcomponents' should probably link to what documentation we have on this, which is in the "Subcomponent Declaration" page